pkg/controller/servicebinding/reconciler.go

```diff
@@ -220,3 +246,21 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	logger.Info("Binding applications with intermediary secret...")
 	return sb.bind()
 }
+
+func updateSBRConditions(dynClient dynamic.Interface, sbr *v1alpha1.ServiceBinding, conditions ...conditionsv1.Condition) error {
```

Comment:
This function is returning an error type, but from where the function is being called, the err is not being handled(if an err is returned).

Reply:
@Avni-Sharma 
There may be error when do the status updating, but in where call this function, seems we do not need check the error.
So your suggestion is it should not return an error right?

Reply:
returning an err is good, but we might need to inform the err to the user